2024/PlatformCon/proposal.md

```diff
@@ -1,19 +1,62 @@
 # Session title
 
+Apparating a Platform with InnerSource
+
 # Description
 
+Fun and engaging story of an amazingly-successful case study of a company-wide platform starting without a funded team.
+
 # Session format
 
+PlatformCon Talk
+
 # Abstract
 
+I was a individual application engineer at Nike with the idea of a company-wide web deployment platform.
+Unable to get funding for a team to build it,
+I partnered, created, and rallied an internal development community that took this idea from skunk works side project to running 88% of all web projects at the company.
+Come for the fun, entertaining, and inspiring story of how it happened and what lessons you can apply in your own dreams of internal development and deployment platforms.
+
 # Format and structure
 
+* Introduction to self and background.
+* Setup: the need for a web deployment platform.
+* Initial rejection of a team to build it.
+* Leverage of community infrastructure to get things going.
+* A 4-team shared web deployment pipeline.
+* More teams come on board and continue to contribute.
+* Formalizing a working team to run the community.
+* Our setup for running Sprints and milestones.
+* Tapping into roadmap planning to expand to new parts of the company.
+* Leveraging metrics to communicate success and find more partners.
+* We become the de-facto company standard, running 88% of all web projects.
+* Our platform strategy spreads to other areas.
+* The InnerSource Commons supports the practice of InnerSource.
+* Lessons learned:
+  * Focus on outcomes.
+  * Build social relationships.
+  * Define contracts.
+  * Run like a startup.
+  * Communicate value.
+
```

Comment:
Looks good, I wonder if this story could tack on "The Future" as a topic at the end mentioning what the ISC is, our partnership with Gartner, and the inevitable rise of InnerSource practices within internal business Platforms?

Reply:
✅